base/expvar.go

```diff
@@ -131,6 +131,8 @@ const (
 	StatKeyPullReplicationsCaughtUp         = "num_pull_repl_caught_up"
 	StatKeyRequestChangesCount              = "request_changes_count"
 	StatKeyRequestChangesTime               = "request_changes_time"
+	StatKeyDcpReceivedCount                 = "dcp_received_count"
+	StatKeyDcpReceivedTime                  = "dcp_received_time"
```

Comment:
Could we make this `dcp_received_total_time` or similar (assuming I've read the code right)? If it's going to be monotonic, having `total` in the name should help allay "why is it taking 3 hours to receive from DCP?!" concerns...

Reply:
We've been using the _time/_count naming pattern across the new stats to indicate two stats that can be combined to calculate a rate or average.  I'd prefer to omit 'total' to avoid naming bloat (since the keys get emitted every time we write stats.

Reply:
We've used `total_xxx` naming for some stats, like `total_auth_time`, which I think makes it clear that they're both cumulative values.

```go
StatKeyAuthSuccessCount = "auth_success_count"
StatKeyAuthFailedCount  = "auth_failed_count"
StatKeyTotalAuthTime    = "total_auth_time"
```

Reply:
I think 'total' in the auth time case is intended to indicate the time combines timings for both the success and failed counts.